src/vaibhav/assignment39/Square.java

```diff
@@ -0,0 +1,29 @@
+package vaibhav.assignment39;
+
+/*Program : 2 
+Create a Square class with field side, one parameterized constructor which accept side value .
+Write a method to find the area of a square. Write a TestSquareClass to verify Square class
+functionality.
+NOTE : square method should not take any parameter, constructor should set side value in
+instance variable and square method should use that instance variable.
+(area=side*side) */
+
+
+public class Square {
+	
+	double side;
+	
+	Square(double side){
+		this.side=side;		
+	}
+	
+	void getArea(){
```

Comment:
as per the method name getArea() you should return the value, Just correcting logically.
if you are giving method name & that starts with get() then you should return value,
if you are giving method name as set() then return type should be void.
just FYI